src/librustdoc/html/static/main.js

```diff
@@ -1062,6 +1072,11 @@
                 type = matches[1].replace(/^const$/, 'constant');
                 query = query.substring(matches[0].length);
             }
+            // In case we just get a '!' as input, we can assume that the user is looking for the
+            // `Never` primitive type.
+            if (raw === '!') {
+                query = 'Never';
+            }
```

Comment:
We could delete this code now right? 😄

Reply:
Oh damn! Completely forgot about it :laughing: